index.js

```diff
@@ -1 +1,12 @@
-// TODO: Start prototyping here
+import esMain from 'es-main';
+import { host, port } from './src/config.js';// eslint-disable-line import/extensions
```

Comment:
It's not good when you have to manually turn off lint at specific lines. I'm pretty sure that you can modify your `.eslintrc` to disable that if you need. And you still need to install a plugin to check imports, see my comment in another PR.